monitoring/cluster_monitoring/configuring-the-monitoring-stack.adoc

```diff
@@ -18,12 +18,25 @@ This section explains what configuration is supported, shows how to configure th
 include::modules/monitoring-maintenance-and-support.adoc[leveloffset=+1]
 include::modules/monitoring-creating-cluster-monitoring-configmap.adoc[leveloffset=+1]
 include::modules/monitoring-configuring-the-cluster-monitoring-stack.adoc[leveloffset=+1]
+
+.Additional resources
+
+* See xref:../../monitoring/cluster_monitoring/configuring-the-monitoring-stack.adoc#creating-cluster-monitoring-configmap_configuring-monitoring[Creating a cluster monitoring ConfigMap] to learn how to create the `cluster-monitoring-config` ConfigMap object.
```

Comment:
the link does not link to `Creating a cluster monitoring ConfigMap` directly

Reply:
@juzhao thank you for your review! This xref works for me, as do all of the other xrefs in this PR. They each take me directly to the module that is listed in the square parenthesis or the top level page if that is intended.

For example, the link in the preceding line navigates in the preview to [Creating a cluster monitoring ConfigMap](https://bz1865805--ocpdocs.netlify.app/openshift-enterprise/latest/monitoring/cluster_monitoring/configuring-the-monitoring-stack.html#creating-cluster-monitoring-configmap_configuring-monitoring).